src/web/jsf/spec/jstl/fntrim/fntrim.xhtml

```diff
@@ -22,7 +22,7 @@
       xmlns:h="http://java.sun.com/jsf/html"
       xmlns:f="http://java.sun.com/jsf/core"
       xmlns:c="http://java.sun.com/jsp/jstl/core"
```

Comment:
Not a blocker (test will pass just fine as Jakarta Faces has currently still backwards compatibility for old XML namespaces), but for the record all those java.sun.com XML namespaces here above should preferably be migrated to the new ones as well. The backwards compatibility may be removed in a future Jakarta Faces version.

Reply:
These are actually jsf test files, I will create a separate PR  to update taglibs for all jsf files remaining. Thanks.